web/src/index.js

```diff
@@ -1 +1,69 @@
-console.log('Hello world !');
+var body = document.body;
+
+var formElt = document.createElement("form");
+var nameInputElt = document.createElement("input");
+var urlInputElt = document.createElement("input");
+var envoyerElt = document.createElement("input");
+
+envoyerElt.type = "submit";
+
+formElt.appendChild(nameInputElt);
+formElt.appendChild(urlInputElt);
+formElt.appendChild(envoyerElt);
+
+body.appendChild(formElt);
+
+formElt.addEventListener("submit", function (e) {
+    //e.preventDefault();
+    var newItem = new FormData();
+    ajaxPost("http://localhost:8081/api/v1/item", {"name":"superpierre", "url":"https://supersite.com"}, function (reponse) {console.log(newItem)}, true);
```

Comment:
About those AJAX calls, we might want to use the [fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) instead of the good old `XMLHttpRequest`.

Three reasons for that : 
- It is way easier to use (for instance error handling)
- It will avoid us some boilerplate code such as `ajaxGet` and `ajaxPost` functions :)
- At the moment we don't really care about browser support

For instance, instead of `ajaxGet("/api/v1/item", function(result) { ... })` you can write `fetch("/api/v1/item").then(function(result) { ... })`, and delete the `ajaxGet` function. Same thing for `ajaxPost`. :)

Reply:
Also for a callback, you might want to use the fat arrow syntax instead of the traditional function keyword.

IE, you can replace:

```
ajaxGet("/api/v1/item", function(result) { ... })
```

by

```
ajaxGet("/api/v1/item", (result) => { ... })
```